doc/xml/user-guide.xml

```diff
@@ -1833,6 +1839,8 @@
 
             <p>Stop the cluster and restore only the test2 database.  Built-in databases (<id>template0</id>, <id>template1</id>, and <id>postgres</id>) are always restored.</p>
 
+            <admonition type="note">Recovery may error unless <br-option>--type=immediate</br-option> is specified. This is because after consistency is reached <postgres/> will flag zeroed pages as errors even for a full-page write. For <postgres/> &amp;ge; <proper>13</proper> the <pg-option>ignore_invalid_pages</pg-option> setting may be used to ignore invalid pages. In this case it is important to check the logs after recovery to ensure that no invalid pages were reported in the selected databases.</admonition>
```

Comment:
This admonition should be type=warn - it is alerting the user to something that can be an issue for them.

Reply:
I thought perhaps that was too strong. But done that way in 320ee4997.